samcli/local/lambdafn/env_vars.py

```diff
@@ -191,7 +193,11 @@ def _stringify_value(self, value):
             result = "false"
 
         # value is a scalar type like int, str which can be stringified
-        else:
+        elif sys.version_info.major > 2:
```

Comment:
Could you add a comment here on why this is needed. It is not super clear on the reason for this by just looking at the code.